src/components/htmlMediaHelper.js

```diff
@@ -131,8 +131,11 @@ import { Events } from 'jellyfin-apiclient';
     }
 
     function setCurrentTimeIfNeeded(element, seconds) {
-        if (Math.abs(element.currentTime || 0, seconds) <= 1) {
-            element.currentTime = seconds;
+        // If it's worth skipping (1 sec or less of a difference)
+        if (Math.abs((element.currentTime || 0) - seconds) <= 1) {
```

Comment:
Looks like we have inverted it - should be `>= 1`.
But looking at this:
https://github.com/jellyfin/jellyfin-web/blob/da2e3d15be6e5ba668b2f2ca2c92fa3486c51149/src/components/htmlMediaHelper.js#L158-L163
I am not sure about which one is right - need to test.
`setCurrentTimeIfNeeded` is currently supposed to seek if `currentTime <= 1` (as you have noticed).

Reply:
Changed back to the correct check

I think this comment applies to the previous check (`element.currentTime === 0 && element.duration >= seconds`), but not sure either.

Reply:
> I think this comment applies to the previous check (`element.currentTime === 0 && element.duration >= seconds`), but not sure either.

Ah, right.

At least with `>= 1` it is logically valid in compare with `Math.abs(element.currentTime || 0) <= 1`, which is `(element.currentTime || 0) <= 1` since `currentTime` is non-negative.